src/elements/common/interactionTargets.js

```diff
@@ -40,6 +40,12 @@ export const ACTIVITY_TARGETS = {
     TASK_APPROVE: 'activityfeed-taskapprove',
     TASK_COMPLETE: 'activityfeed-taskcomplete',
     TASK_REJECT: 'activityfeed-taskreject',
+    TASK_OPTIONS: 'activityfeedtaskoptions', // component
```

Comment:
These all get built down into strings following the format (usually) of `page-component-feature-target`, so there may end up being a lot of duplicated info in these strings, such as `files-activityfeedtaskoptions--activityfeed-taskoptions-delete`